src/main/resources/application-local.yml

```diff
@@ -1,6 +1,6 @@
 oauth:
   client:
-    id: whereabouts-api-client
+    id: accredited-programmes-api
```

Comment:
Does this work locally, or does it need to be `hmpps-accredited-programmes-client-1` as in `application.yml`?

Reply:
Having tested this yesterday, we believe this works locally, right?

Reply:
yes, this works. However we've changed it to `hmpps-accredited-programmes-client-1`

Reply:
ah can you amend this last commit and change it to that value?

Reply:
Looks like we have already done it, the comment was from an initial commit its `hmpps-accredited-programmes-client-1` now

Reply:
I can still see this in this file, are you sure you're not looking at `application.yml` (rather than `application-local.yml`?